### ISSUE TITLE
MediaMTX

### ISSUE THREAD

**Issue by DasBrin** [community-scripts/ProxmoxVE#899]
### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

MediaMTX

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/mediamtx.sh)"

### 📝 Provide a clear and concise description of the issue.

Creation of MediaMTX LXC Fails with a error regardless if Default or Advanced Settings if BTRFS is used.
On a ZFS PVE is works.
 
```
unable to create CT 100 - internal error: bad disk name: debian-12-standard_12.7-1_amd64.tar.zst at /usr/share/perl5/PVE/Storage/BTRFSPlugin.pm: 200
bash: line 201: exit: A problem occured while trying to create container.: numeric argument required
```

### ⚙️ What settings are you using?

- [X] Default Settings
- [X] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

😐 Have BTRFS PVE.
🙂 Run Script.
🥳 Error 

### ❌ Paste the full error output (if available).

```
    __  ___         ___       __  __________  __
   /  |/  /__  ____/ (_)___ _/  |/  /_  __/ |/ /
  / /|_/ / _ \/ __  / / __ `/ /|_/ / / /  |   / 
 / /  / /  __/ /_/ / / /_/ / /  / / / /  /   |  
/_/  /_/\___/\__,_/_/\__,_/_/  /_/ /_/  /_/|_|  
                                                
  🧩  Using Advanced Settings
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  🔐  Root Password: Automatic Login
  🆔  Container ID: 100190
  🏠  Hostname: mediamtx
  💾  Disk Size: 4
  🧠  CPU Cores: 2
  🛠️  RAM Size: 2048
  🌉  Bridge: vmbr100
  📡  IP Address: 10.10.100.190/24
  🌐  Gateway IP Address: 10.10.100.1
  📡  APT-Cacher IP Address: 10.10.100.250
  🚫  Disable IPv6: yes
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  🔑  Root SSH Access: no
  🔍  Verbose Mode: yes
  🚀  Creating a MediaMTX LXC using the above advanced settings
  ✔️  Using btrfs-hdd for Template Storage.
  ✔️  Using btrfs-ssd for Container Storage.
  ✔️  Updated LXC Template List
 unable to create CT 100190 - internal error: bad disk name: debian-12-standard_12.7-1_amd64.tar.zst at /usr/share/perl5/PVE/Storage/BTRFSPlugin.pm: 200
bash: line 201: exit: A problem occured while trying to create container.: numeric argument required
```

```
    __  ___         ___       __  __________  __
   /  |/  /__  ____/ (_)___ _/  |/  /_  __/ |/ /
  / /|_/ / _ \/ __  / / __ `/ /|_/ / / /  |   / 
 / /  / /  __/ /_/ / / /_/ / /  / / / /  /   |  
/_/  /_/\___/\__,_/_/\__,_/_/  /_/ /_/  /_/|_|  
                                                
  ⚙️  Using Default Settings (  🔍   Verbose)
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  💾  Disk Size: 4GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 2048MB
  🆔  Container ID: 100
  🔍  Verbose Mode: Enabled
  🚀  Creating a MediaMTX LXC using the above default settings
  
  ✔️  Using btrfs-hdd for Template Storage.
  ✔️  Using btrfs-hdd for Container Storage.
  ✔️  Updated LXC Template List
 unable to create CT 100 - internal error: bad disk name: debian-12-standard_12.7-1_amd64.tar.zst at /usr/share/perl5/PVE/Storage/BTRFSPlugin.pm: 200
 bash: line 201: exit: A problem occured while trying to create container.: numeric argument required
```

**Comment by MickLesk** [comment]
its an issue of your proxmox. It cant be download the debian zst from pveam available

**Comment by DasBrin** [comment]
@ Future Anybody with that issue.
Issue in "libpve-storage-perl 8.3.1" , Fixed in "libpve-storage-perl 8.3.2"
